test/commands/env/create/sandboxCreate.test.ts

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+import * as os from 'os';
+import * as path from 'path';
+import {
+  AuthInfo,
+  GlobalInfo,
+  Lifecycle,
+  Messages,
+  Org,
+  SandboxEvents,
+  SandboxProcessObject,
+  SandboxUserAuthResponse,
+  SfError,
+  SfInfo,
+  SfInfoKeys,
+  SfOrg,
+  SfOrgs,
+  SfProject,
+} from '@salesforce/core';
+import { Progress } from '@salesforce/sf-plugins-core';
+import { Config as IConfig } from '@oclif/core/lib/interfaces';
+import { fromStub, stubInterface, stubMethod } from '@salesforce/ts-sinon';
+import * as sinon from 'sinon';
+import { expect } from 'chai';
+import { assert } from 'sinon';
+import { Ux } from '@salesforce/sf-plugins-core/lib/ux';
+import CreateSandbox from '../../../../src/commands/env/create/sandbox';
+import { getSandboxProgress } from '../../../../src/shared/sandboxReporter';
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-env', 'create.sandbox');
+
+const sandboxProcessObj: SandboxProcessObject = {
+  Id: '0GR4p000000U8EMXXX',
+  Status: 'Completed',
+  SandboxName: 'TestSandbox',
+  SandboxInfoId: '0GQ4p000000U6sKXXX',
+  LicenseType: 'DEVELOPER',
+  CreatedDate: '2021-12-07T16:20:21.000+0000',
+  CopyProgress: 100,
+  SandboxOrganization: '00D2f0000008XXX',
+  SourceId: '123',
+  Description: 'sandbox description',
+  ApexClassId: '123',
+  EndDate: '2021-12-07T16:38:47.000+0000',
+};
+
+const fakeOrg: SfOrg = {
+  orgId: '00Dsomefakeorg1',
+  instanceUrl: 'https://some.fake.org',
+  username: 'somefake.org',
+};
+
+const fakeSfInfo: SfInfo = {
+  [SfInfoKeys.ORGS]: { [fakeOrg.username]: fakeOrg } as SfOrgs,
+  [SfInfoKeys.TOKENS]: {},
+  [SfInfoKeys.ALIASES]: { testProdOrg: fakeOrg.username },
+};
+
+describe('org:create', () => {
+  beforeEach(() => {
+    GlobalInfo.clearInstance();
+    stubMethod(sandbox, GlobalInfo.prototype, 'read').callsFake(async (): Promise<SfInfo> => {
+      return fakeSfInfo;
+    });
+    stubMethod(sandbox, GlobalInfo.prototype, 'write').callsFake(async (): Promise<SfInfo> => {
+      return fakeSfInfo;
+    });
+  });
+
+  process.env.SF_SFDX_INTEROPERABILITY = 'false';
```

Comment:
better to stub env inside the test than to have other tests impacted by unexpected env mutations [I know we don't run UT in parallel because mocha but some of the problems with toolbelt UT came from env mutations that persist beyond a single test]

Reply:
That was left over from one of the many experiments I was running. I will remove. Thanks for catching this.